libsupport/src/ArrowVisitor.cpp

```diff
@@ -148,7 +148,8 @@ struct ToArrayVisitor : public katana::ArrowVisitor {
 
   using ResultType = katana::Result<std::shared_ptr<arrow::Array>>;
 
-  using AcceptTypes = std::tuple<katana::AcceptAllArrowTypes>;
+  using AcceptTypes = std::tuple<katana::tuple_cat_t<
+      katana::AcceptAllArrowTypes, std::tuple<arrow::FixedSizeBinaryType>>>;
```

Comment:
NOTE: arrow::FixedSizeBinary should be rolled into AcceptAllArrowTypes eventually, but that touches most visitors, so I'm dragging my feet on it